saleor/account/migrations/tasks/saleor3_13.py

```diff
@@ -0,0 +1,23 @@
+from django.db.models import Q, Exists, OuterRef
+
+from ....celeryconf import app
+from ...models import User, Group
+
+# The batch of size 5000 took about 0.5 s to assign users to groups
+BATCH_SIZE = 5000
+
+
+@app.task
+def create_full_channel_access_group_task(group_name):
```

Comment:
I would double-check if this task should go to 3.13 or maybe to 3.12. If you will merge this task to 3.13, then we will have a moment, when the data will not be migrated correctly, which means, that you could block the access to dashboard.

Reply:
Thanks! Yes, it probably should go to 3.12. I will handle it after merging to feature branch